### PR TITLE
Players should be able to resume an in progress Tic Tac Toe game.

### PR DESCRIPTION
**Pull request by WasanMoosa**
file

### CODE REVIEW

File: TICTAC/src/TICTACWithResumeFile.java [hunk 1]
```diff
@@ -1,80 +1,176 @@
...
+			if (myFile.exists()) {
+				Scanner scfile = new Scanner(myFile);
+				System.out.print("Do you want to resume (press 0) ? or Start new game (press 1)");
+				int state = sc.nextInt();
+				if (state == 0) {
+
+					continueBoolean = true;
+					int i = 0; // To seperate the file (0-2) for position, (3)for symbol, (4)for count
+					while (scfile.hasNextLine()) {
+						if (i < 3) {
```
Comment (BaraahAlKalbani) [inline TICTAC/src/TICTACWithResumeFile.java:46]: use meaningful var names instead of i and q

**Review by BaraahAlKalbani — CHANGES_REQUESTED** [review]
Great code by the way but change the some variables names to meaningful ones

File: TICTAC/src/TICTACWithResumeFile.java (hunk 1 above)
Comment (WasanMoosa) [reply TICTAC/src/TICTACWithResumeFile.java:46]: Okay

**Review by BaraahAlKalbani — APPROVED** [review]

File: TICTAC/src/TICTACWithResumeFile.java [hunk 2]
```diff
@@ -1,80 +1,176 @@
...
+
 				}
-				
+
 			}
-		 System.out.print("The End!");
-		 }
-	
 
-//////////////////////////////////////////////////////////////////////////////////////////////////////////////
```
Comment (FatemaAlbalushi) [inline TICTAC/src/TICTACWithResumeFile.java:75]: Try to generate java dox.

File: TICTAC/src/TICTACWithResumeFile.java [hunk 3]
```diff
@@ -88,39 +184,83 @@ static void shownBox() {
 		}
 	}
```
Comment (FatemaAlbalushi) [inline TICTAC/src/TICTACWithResumeFile.java:190]: Remove the unused spacing

File: TICTAC/src/TICTACWithResumeFile.java [hunk 4]
```diff
@@ -88,39 +184,83 @@ static void shownBox() {
...
-	static int[] ChoosePosition (char move, char symbol[]) {
+
+	/**
+	 * Method to choose the position return the number place of the chosen position
+	 */
+	static int[] ChoosePosition(char move, char symbol[]) {
 
 		int moveInt = Character.getNumericValue(move);
-		int i = (int) Math.ceil(moveInt/3.0)-1;
-		int j= moveInt%3;
```
Comment (FatemaAlbalushi) [inline TICTAC/src/TICTACWithResumeFile.java:105]: Instead of using i,j try to use meaningful name.

**Review by FatemaAlbalushi — COMMENTED** [review]
Great work.

**Review by FatemaAlbalushi — APPROVED** [review]

**Review by AnwaarAlbusaidi — APPROVED** [review]
great work

**Pull request merged**